src/prop/cadical.cpp

```diff
@@ -204,6 +204,26 @@ unsigned CadicalSolver::getAssertionLevel() const
 
 bool CadicalSolver::ok() const { return d_inSatMode; }
 
+void CadicalSolver::setDrat()
+{
+  sprintf(d_tempDratFilePath, "dratProof_%d.drat", getpid());
+  d_dratFile = fopen(d_tempDratFilePath, "wb");
```

Comment:
When is the tempfile deleted?

Reply:
Currently it's not being deleted

Reply:
Yes, ideally the temporary file would be created and cleaned up RAII-style in CadicalSolver, but for now deleting in the destructor should be ok.